.github/workflows/pull-request.yml

```diff
@@ -23,12 +23,6 @@ jobs:
                     name: Ballerina Internal Log
                     path: graphql-ballerina/ballerina-internal.log
                     if-no-files-found: ignore
-            -   name: Archive Code Coverage JSON
```

Comment:
Do we need to remove this? IMO, keeping this will help developers get the exact lines for the coverage since we would not produce that info with jacoco report.